components/tools/OmeroPy/test/integration/clitest/test_pyramids.py

```diff
@@ -98,9 +98,24 @@ def import_pyramid(self, tmpdir, name=None):
         assert id >= 0
         time.sleep(30)
         query_service = self.client.sf.getQueryService()
+        param = ParametersI().addId(id)
+        if settings:
+            tb = self.client.sf.createThumbnailStore()
+            try:
+                tb.getThumbnailByLongestSideSet(rint(16), [id])
+            finally:
+                tb.close()
+            # check the rendering settings
+            query = """
+            select rdef from RenderingDef as rdef
+            left outer join fetch rdef.pixels as p
+            where p.id =:id
+            """
+            rnd_defs = query_service.findAllByQuery(query, param)
```

Comment:
Is this happening in all-groups context or is that not an issue in this case?

Reply:
method is admin only

Reply:
I may be misunderstanding. I need the `context` variable to get a result from this script:

```python
#!/usr/bin/env python

import omero
from omero.gateway import BlitzGateway

conn = BlitzGateway('root', 'omero', host='eel.openmicroscopy.org')
conn.connect()
query_service = conn.getQueryService()

image_id = 25956

query = 'SELECT name FROM Image WHERE id = :id'
params = omero.sys.ParametersI()
params.addId(omero.rtypes.rlong(image_id))
context = {'omero.group': '-1'}

image_names = query_service.projection(query, params, context)

for image_name in image_names:
    print(image_name[0].val)

conn._closeSession()
```